mlflow/spacy.py

```diff
@@ -0,0 +1,194 @@
+"""
+The ``mlflow.spacy`` module provides an API for logging and loading spaCy models.
+This module exports spacy models with the following flavors:
+
+Spacy (native) format
```

Comment:
spaCy

Reply:
This one I changed just the one on line 5 and kept the "mlflow.spacy" as is as it references the module itself. If that makes sense to you

Reply:
Yes, `mlflow.spacy` should be kept as is.